src/components/home/student/AddStudent.js

```diff
@@ -0,0 +1,401 @@
+import {
+  Button,
+  Divider,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+} from "@mui/material";
+import { useFormik } from "formik";
+import React, { useEffect, useState } from "react";
+import { ADD_STUDENT_URL, BRANCH_URL, COURSE_URL } from "../../../constants/apiEndpoints";
+import http_lib from "../../../http/http";
+import { addStudentSchema } from "../../../schemas/addStudentSchema";
+import "../style.css";
+import { AttachFile, CloudUpload } from "@mui/icons-material";
+import BulkUploadNote from "../../bulkUploadNote/bulkUploadNote";
+import { useNavigate } from "react-router-dom";
+
+const styles = {
+  input: {
+    display: "none",
+  },
+  button: {
+    margin: "8px",
+  },
+};
+
+const initialValues = {
+  enrollment_number: "",
+  name: "",
+  section: "",
+  batch: "",
+  class_roll_number: "",
+  student_group: "",
+  student_phone_number: "",
+  student_email_id: "",
+  mother_name: "",
+  father_name: "",
+  parent_phone_number: "",
+  branch: "",
+  course: "",
+  file: "",
+  fileUploadInput: "",
+};
+
+const AddStudent = () => {
```

Comment:
```suggestion
const AddStudent = ({ setLoader }) => {
```

pls make sure to show loader if making any api call